packages/shared/src/components/drawers/Drawer.tsx

```diff
@@ -50,6 +50,7 @@ export interface DrawerProps
   onClose: PopupCloseFunc;
   displayCloseButton?: boolean;
   appendOnRoot?: boolean;
+  instantOpen?: boolean;
```

Comment:
Added new prop to the drawer to instantly show it, and now animate it in. As in this case, we've already rendered a full screen modal, so it doesn't make much sense to animate in the new drawer.